notify-and-hydrate-state/pr-files.go

```diff
@@ -12,35 +13,88 @@ import (
 	"sigs.k8s.io/yaml"
 )
 
+type PrFiles struct {
+	AddedModified []string
+	Deleted       []string
+}
+
 func (m *NotifyAndHydrateState) GetPrChangedFiles(
 
 	ctx context.Context,
 
-	ghRepo string,
-
-	prNumber string,
+	claimsRepo *Directory,
 
-) ([]string, error) {
+) (PrFiles, error) {
 
-	command := strings.Join([]string{"pr", "view", prNumber, "--json", "files", "-R", ghRepo}, " ")
+	result := PrFiles{
+		AddedModified: []string{},
+		Deleted:       []string{},
+	}
 
-	content, err := dag.Gh().Run(ctx, m.GhToken, command, GhRunOpts{DisableCache: true})
+	c, err := dag.
+		Container().
+		From("alpine/git").
+		WithMountedDirectory("/repo", claimsRepo).
+		WithWorkdir("/repo").
+		Sync(ctx)
+
+	amResp, err := c.
+		WithExec([]string{
+			"diff",
+			"origin/" + m.ClaimsDefaultBranch,
+			"-M90%",
+			"--name-only",
+			"--diff-filter=d",
+		}).
+		Stdout(ctx)
 
 	if err != nil {
 
-		panic(err)
+		return result, err
+
+	}
+
+	for _, line := range strings.Split(amResp, "\n") {
+
+		if line == "" {
+
+			continue
+
+		}
+
+		result.AddedModified = append(result.AddedModified, line)
```

Comment:
But the `--diff-filter=D` should output the deleted files only :thinking: 
And the `--diff-filter=AD` the files added or modified